src/sentry/api/endpoints/organization_tags.py

```diff
@@ -29,6 +29,13 @@ def get(self, request: Request, organization) -> Response:
                     # Defaults to True, because the frontend caches these tags globally
                     include_transactions=request.GET.get("include_transactions", "1") == "1",
                 )
+                if not features.has(
+                    "organizations:javascript-console-error-tag",
+                    organization,
+                    actor=None,
+                ):
+                    results = [tag for tag in results if tag != "empty_stacktrace.js_console"]
```

Comment:
Is this so the tag does *not* show up on the list of tags to pick from?

Reply:
yes, we don't want users from other orgs to see the tag.